doc/3-pattern/README.md

```diff
@@ -38,15 +38,33 @@ clicking on a channel name mutes that channel.
 
 double-clicking or right-clicking it enables solo mode, in which only that channel will be audible.
 
-clicking the `++` at the top left corner of the pattern view cycles through three channel bar view modes:
-- **Compact**: shows only channel names.
-- **Expanded**: as shown above. adds buttons:
+clicking the `++` at the top left corner of the pattern view pops up a small dialog to set view modes:
+- **Effect columns/collapse**: as shown above. adds buttons:
   - **-**: collapse visible columns. changes to **+** when columns are hidden; click to expand them.
   - **<**: disables the last effect column and hides it. effects are not deleted...
   - **>**: adds an effects column. if one previously existed, its contents will be preserved.
 - **Pattern names**: displays pattern names (per channel). pattern names are also visible when hovering over a pattern in the order list.
-
-right-clicking the `++` toggles the visualizer, which is active only during playback.
+- **Channel group hints**: display indicators beneath the names of channels that function together in some way.
+- **Visualizer**: during playback, show visual effects depending on activity in each channel.
```

Comment:
"show particles in the pattern view".

Reply:
Are you sure? There's more than particle effects going on (thinking of pitch bends).

Reply:
"show visual effects in the pattern view" maybe?